packages/embeddings/tests/Unit/Generator/EmbeddingGeneratorTest.php

```
@@ -82,7 +82,7 @@ public function testGenerateEmbeddingWithHeaderGenerator(): void
 
         $headerText = 'Generated header: ';
 
-        $headerGenerator = fn (EmbeddingInterface $embedding) =>
+        $headerGenerator = static fn (EmbeddingInterface $embedding) =>
```

PHPMD (2.15.0)

Avoid unused parameters such as '$embedding'. (undefined)

(UnusedFormalParameter)